utils/pbconvert/pbconvert.go

```diff
@@ -45,42 +68,48 @@ func InstanceFilterToPB(filter cloudprotocol.InstanceFilter) *pb.InstanceIdent {
 	return ident
 }
 
-func InstanceIdentToPB(ident aostypes.InstanceIdent) *pb.InstanceIdent {
-	return &pb.InstanceIdent{ServiceId: ident.ServiceID, SubjectId: ident.SubjectID, Instance: int64(ident.Instance)}
-}
-
-func NetworkParametersToPB(params aostypes.NetworkParameters) *pb.NetworkParameters {
-	networkParams := &pb.NetworkParameters{
-		Ip:         params.IP,
-		Subnet:     params.Subnet,
-		VlanId:     params.VlanID,
-		DnsServers: make([]string, len(params.DNSServers)),
-		Rules:      make([]*pb.FirewallRule, len(params.FirewallRules)),
+// NewInstanceIdentFromPB converts InstanceIdent from protobuf to AOS type.
+func NewInstanceIdentFromPB(ident *pbcommon.InstanceIdent) aostypes.InstanceIdent {
+	return aostypes.InstanceIdent{
+		ServiceID: ident.GetServiceId(),
+		SubjectID: ident.GetSubjectId(),
+		Instance:  ident.GetInstance(),
 	}
+}
 
-	copy(networkParams.GetDnsServers(), params.DNSServers)
+// InstanceIdentToPB converts InstanceIdent from AOS type to protobuf.
+func InstanceIdentToPB(ident aostypes.InstanceIdent) *pbcommon.InstanceIdent {
```

Comment:
It's not clear why some functions contain New prefix, some doesn't

Reply:
It was done before, I just follow the pattern. Ok, will rename.